### ISSUE TITLE
can't install the firmware "no such file or directory"

### ISSUE THREAD

**Issue by minijouin** [googlecreativelab/open-nsynth-super#35]
Hi ! 

I'm trying to install the firmware on the raspberry.
Nothing happens when I run "sudo mount -o remount,rw /"
And if I try to run "cd home/pi/open-nsynth-super-master/firmware/src" then it says "No such file or directory".

I tried already to set the image on the micro SD again, but nothing changed.

Does anyone know ?

**Comment by KyleWerle** [comment]
That remount doesn't give any confirmation, but it allows you to write files to your file system.

I think the guide is kind of following if you were using git to download the repository, which downloads the folder with -master at the end.

Try to use 
``cd home/pi/open-nsynth-super/firmware/src
``

That should be the correct path.

You can search around using 
``cd home
``
``ls
``
``cd pi
``
``ls
``
``cd open-nsynth-super
``
``ls
``

This is how you can navigate around using the command line.

Good luck!

**Comment by minijouin** [comment]
Yes it worked ! 
But then I ran "make install" and it failed... Do you know why ? 

The code is ending with 
"makefile:40: recipe for target "install" failed
make: *** (install) error 1"

**Comment by minijouin** [comment]
Target name : stm32f0x.cpu 
Type : cortex_m
Endian : little
TapName : stm32f0x.cpu 
State : unknown

Is it related to the microchip stm32f030k6 ? What does .cpu mean ?

**Comment by minijouin** [comment]
The error seems to be "Could not initialize the debug port" with that target

**Comment by minijouin** [comment]
Error : Could not initialize the debug port
Target Name : stm32j0x.cpu
Type : Cortex_m
Endian : little
TapName : stm32f0x.cpu
State : unknown

adaptater speed : 101 kHz
Error : Could not initialize the debug port
Error : Target not examined, will not halt after reset ! 
TARGET : stm32f0x.cpu -  Not Halted
In procedure "reset" called at file "openocd.cfg", line 22
In procedure "ocd_bouncer"

Make file :40: recipe for target "install" failed
make: *** (install) error 1

PLEASE HELP ME ! :D

**Comment by KyleWerle** [comment]
Sounds like a hardware issue. Check your solder joints on the STM? Big square chip. Best to just go through all your solder joints.

**Comment by minijouin** [comment]
Does anyone know how the STM should be placed ? Where does the dot should be placed exactly ? 
Because I actually think I've soldered it in the wrong direction.

**Comment by elimuro** [comment]
the orientation is shown in the [images](https://github.com/googlecreativelab/open-nsynth-super/blob/master/images/ons_pcb_mcu.jpg) where you can see the dot fairly clearly

**Comment by minijouin** [comment]
Yes I know, I soldered it as it was on the images, but then I looked at the schematics, and into renders (>open-nsynth-super-bottom.pos) and it's written that U7 has to be 90°, compared to C1 which is 0°...

So should I try to turn it 90° ?

**Comment by minijouin** [comment]
@elimuro did you find any solution ? I turned STM32f0x of 90° but I've got the same problem, still...

**Comment by minijouin** [comment]
Hum ok, but in renders U7 has to be 90°, just like R21 to R31. And "U7" is written in white on the board at 90° as well.
So I turned it but the problem is still the same !

**Comment by KyleWerle** [comment]
I'm not sure how much experience you have with SMT electronics, but just removing an old chip and replacing it is not quite easy. To flip it over and over again seems like a pretty quick way to destroy your ARM board.

The location of the dot on the image is the same as the boards I have that are working.

**Comment by furynolo** [comment]
Was anybody able to make any headway on this?  I am also having this problem and haven't been able to successfully troubleshoot it yet.  Thanks!

**Comment by minijouin** [comment]
Hi, 

I just checked the C7 capacitor with a multimeter and it says 0,009 V but 47 ohms... Depending on the mode I'm measuring with.

Any clue ?

**Comment by notthetup** [comment]
Yea. The STM32 doesn't have any power. You might want to check the 3.3V power supply side of things.

**Comment by kaosbeat** [comment]
seems related to #67 
I checked the power across C7, I get a steady 5.15V
is that too much then?
